docs/STEPS.md

```diff
@@ -116,6 +117,21 @@ Support
 |:-----:|:--------------:|:----------:|
 |   ✔   |       ✔        |     ✔      |
 
+## Wait for window \<secs> and save handle as \<placeholder>
+
+> \* Wait for window "2" and save handle as "handles"
+
+> \* Wait for window "0.3" and save handle as "handles"
+
+> \* Wait for window "\${secs}" and save handle as "handles"
+
+Waits for a window the specified time in seconds. If there is a new window after the time expired, then the window handle is been saved, using the value of the placeholder. You have to save all window handles before, see also: [Save window handles as <placeholder>](#save-window-handles-as-placeholder). 
```

Comment:
Could you explain the use case a little more? I think in between the first window handle step and this the user would want to do some stuff that creates a new window somehow?